### ISSUE TITLE
Upstream?

### ISSUE THREAD

**Issue by eszlari** [refi64/zypak#7]
Is there a plan to upstream this project into Chromium at some point in the future?

**Comment by refi64** [comment]
The Chromium Flatpak uses patches for this, but at the moment upstream has
said they're not interested in adding new sandboxes, and if they were to
ever gain interest in it, it probably wouldn't be right now due to the
global situation.

**Comment by eszlari** [comment]
Do you think that breakage due to upstream changes is likely? Would be too bad, if it suddenly breaks like the Steam flatpak (https://github.com/flathub/com.valvesoftware.Steam/issues/642), because of lack of upstream involvement.

**Comment by refi64** [comment]
I don't think so, most of the Zygote code is pretty stable at this point, and the setuid sandbox will stick around for a long time, since because Debian still doesn't have user namespaces on by default, and even if they enabled it, Chromium would still have to leave it until the last Debian release without it reaches EOL.

**Comment by eszlari** [comment]
Thanks for the clarification! This sounds reassuring.

**Comment by DemiMarie** [comment]
Would it be possible to remove the LD_PRELOAD hacks, and have the zypak binary be the setuid sandbox?
